app/src/main/java/org/oppia/app/utility/Logger.kt

```diff
@@ -0,0 +1,197 @@
+package org.oppia.app.utility
+
+import android.util.Log
+import org.oppia.app.application.OppiaApplication
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+import java.io.IOException
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+
+/** Wrapper class for Android Log. */
+object Logger {
+
+  private val ENABLE_CONSOLE_LOG = true
+  private val ENABLE_FILE_LOG = true
+  private val GLOBAL_LOG_LEVEL = LogLevel.VERBOSE
+  private val LOG_DIRECTORY =
```

Comment:
We could make this a ``File`` instead and rely on system behavior to form the path:

``private val LOG_DIRECTORY = File(context.filesDir, "oppia_app.log")``

Also note that I switched it to files dir because per https://developer.android.com/reference/android/content/Context.html#getExternalFilesDir(java.lang.String) the external files directory may not always be available, and it can be accessed by any application. We shouldn't let the log file be open access since it may contain sensitive information.

Reply:
Changed code to `private val LOG_DIRECTORY = File(context.filesDir, "oppia_app.log")`
to store files internally